### PR TITLE
Add `teleport_proxy_db_active_connections_total` gauge.

### PR DESCRIPTION
**Pull request by Tener**
See also: https://github.com/gravitational/teleport/issues/26667.

### CODE REVIEW

File: lib/srv/db/proxyserver.go [hunk 1]
```diff
@@ -516,6 +516,16 @@ func (s *ProxyServer) Proxy(ctx context.Context, proxyCtx *common.ProxyContext,
 		return trace.Wrap(err)
 	}
 
+	var labels prometheus.Labels
+	if len(proxyCtx.Servers) > 0 {
+		labels = getLabelsFromDb(proxyCtx.Servers[0].GetDatabase())
```
Comment (zmb3) [inline lib/srv/db/proxyserver.go:521]:
```suggestion
		labels = getLabelsFromDB(proxyCtx.Servers[0].GetDatabase())
```

https://github.com/golang/go/wiki/CodeReviewComments#initialisms

File: docs/pages/includes/metrics.mdx [hunk 2]
```diff
@@ -109,6 +109,7 @@
 | `teleport_proxy_db_connection_dial_failures_total`     | counter   | Teleport Proxy | Number of failed dial attempts from Proxy to DB service made.                                                                            |
 | `teleport_proxy_db_attempted_servers_total`            | histogram | Teleport Proxy | Number of servers processed during connection attempt to the DB service from Proxy service.                                              |
 | `teleport_proxy_db_connection_tls_config_time_seconds` | histogram | Teleport Proxy | Time to fetch TLS configuration for the connection to DB service from Proxy service.                                                     |
+| `teleport_proxy_db_active_connections_total`           | gauge     | Teleport Proxy | Number of currently active connections to DB service from Proxy service.                                                                 |
```
Comment (zmb3) [inline docs/pages/includes/metrics.mdx:112]: Is this metric actually exposed by the proxy service or the db service? I thought `lib/srv/db` was the database agent?

Comment (Tener) [reply docs/pages/includes/metrics.mdx:112]: Not entirely, which is a sad thing. `NewProxyServer` from `lib/srv/db/proxyserver.go` is used in the proxy service.

**Review by zmb3 — APPROVED** [review]

File: lib/srv/db/proxyserver.go [hunk 3]
```diff
@@ -516,6 +516,16 @@ func (s *ProxyServer) Proxy(ctx context.Context, proxyCtx *common.ProxyContext,
 		return trace.Wrap(err)
 	}
 
+	var labels prometheus.Labels
+	if len(proxyCtx.Servers) > 0 {
+		labels = getLabelsFromDB(proxyCtx.Servers[0].GetDatabase())
+	} else {
+		labels = getLabelsFromDB(nil)
+	}
```
Comment (greedy52) [inline lib/srv/db/proxyserver.go:524]: I don't think it's possible to have empty `proxyCtx.Servers` here. but good to check anyway xD

**Review by greedy52 — APPROVED** [review]

**Pull request merged**